src/config/index.js

```diff
@@ -0,0 +1,18 @@
+import mongoose from 'mongoose';
```

Comment:
Hi @OlutobiOgunsola, really good job on your progress here.

I meant you should move this to `/src/config/database.js`. `/src/config/index.js` will be a different file.

- Please copy all your code here and paste in an `/src/config/database.js` file so that we can move to the next step from there.

Reply:
done